python/ray/train/tests/test_huggingface_trainer.py

```diff
@@ -79,10 +76,14 @@ def test_e2e(ray_start_4_cpus, save_strategy):
     assert result.metrics["epoch"] == 4
     assert result.metrics["training_iteration"] == 4
     assert result.checkpoint
+    assert "eval_loss" in result.metrics
 
     trainer2 = HuggingFaceTrainer(
         trainer_init_per_worker=train_function,
-        trainer_init_config={"epochs": 5},  # this will train for 1 epoch: 5 - 4 = 1
+        trainer_init_config={
+            "epochs": 5,
+            "save_strategy": save_strategy,
```

Comment:
is save_strategy supposed to be use somewhere? doesn't seem like we're using it in the train_function.

Reply:
It's used in the training function, see line 66